static/scripts/rewards/cirip/query-reverse-ens.ts

```diff
@@ -1,29 +1,26 @@
-import { app } from "../app-state";
-import { useHandler, useRpcHandler } from "../web3/use-rpc-handler";
 import { ethers } from "ethers";
+import { useHandler } from "../web3/use-rpc-handler";
 
-export async function queryReverseEns(address: string, networkId: number) {
+export async function queryReverseEns(address: string) {
   // Try to get the ENS name from localStorage
   const cachedEnsName = localStorage.getItem(address);
-  const endpoint = app.provider?.connection.url || (await useRpcHandler(app)).connection.url;
 
-  if (!endpoint) {
-    console.error("ENS lookup failed: No endpoint found for network ID", networkId);
-    if (cachedEnsName) return cachedEnsName;
+  const handler = useHandler(1);
+  // todo fix .getFirstAvailableRpcProvider() can return wss:// in error
```

Comment:
does wss really give error? I remember making a provider with wss from rpc-handler and it was working without problem. In fact, it was much faster that HTTP ones.

Reply:
I can't remember exactly what the error was mate but it looked and felt like it had returned a wss provider in error not that it had made a wss connection and the call failed